src/commands/DebuggerCommands.ts

```diff
@@ -68,7 +67,16 @@ export namespace DebuggerCommands {
       txHash = txHashSelection.detail || txHashSelection.label;
     }
 
-    await startDebugging({txHash, workingDirectory, providerUrl, disableFetchExternal: false});
+    // Sets the parameters for the debug session
+    const args = {
+      txHash,
+      workingDirectory,
+      network: debugNetworkOptions?.name,
+      disableFetchExternal: false,
+    } as DebuggerTypes.DebugArgs;
```

Comment:
Do we need this type assertion here?

Reply:
I put only to make explicit the type, but no problem in remove that